lib/compiler/src/graph.cc

```diff
@@ -14,16 +14,15 @@
  */
```

Comment:
Should this file be removed? I'm guessing it's just a modified version of the old code that's now outdated?